requirements-web.in

```diff
@@ -6,4 +7,5 @@ greenlet>=1.1.0
 psycopg2-binary
 pydantic
 SQLAlchemy<1.5
+swagger-ui-bundle
```

Comment:
Connexion is installed with:

    pip install connexion[swagger-ui]

So we need it so that there are no problems with the swagger. 
(Or I think we can also disable the swagger, but I find it quite useful, to be honest.)

Reply:
Oh, so is this like a replacement for [the API docs](https://containerbuildsystem.github.io/cachito/)? Or something different? https://github.com/spec-first/connexion#the-swagger-ui-console

Maybe this deserves a separate PR with a docs update?

Reply:
Yeah, connexion includes the swagger-ui by default (it can also be disabled). It's pretty similar to the API docs we currently have with redoc, so we should probably pick one or the other.

Reply:
Should we disable it for now and, if we want it, do it in a follow-up PR/story?

Reply:
please don't remove swagger :)

Reply:
So we keep both documentations? http://localhost:8080 and http://localhost:8080/api/v1/ui ?

Maybe with a follow-up story to replace the root page with http://localhost:8080/api/v1/ui, or at least document that the latter one exists?

Reply:
well, keep it at current location, and for any other changes, open a new story